picker/Rofimoji.py

```diff
@@ -173,36 +171,26 @@ def resolve_all_files(self):
         file_names = self.args.files
 
         if len(file_names) == 1 and file_names[0] == 'all':
-            file_names = [os.path.splitext(file)[0] for file in
-                          os.listdir(os.path.join(os.path.dirname(__file__), "data"))
-                          if fnmatch.fnmatch(file, "*.csv")]
+            file_names = [file.stem for file in (Path(__file__).parent / "data").glob("*.csv")]
         return file_names
 
-    def load_from_file(self, file_name: str) -> str:
-        provided_file = os.path.join(os.path.dirname(__file__), "data", file_name + '.csv')
-        if os.path.isfile(file_name):
+    def load_from_file(self, file_name: Path) -> str:
+        provided_file = Path(__file__).parent / "data" / f"{file_name}.csv"
+        if file_name.is_file():
             actual_file_name = file_name
-        elif os.path.isfile(provided_file):
+        elif provided_file.is_file():
             actual_file_name = provided_file
         else:
             raise FileNotFoundError(f"Couldn't find file {file_name}")
 
-        with open(actual_file_name, "r") as file:
-            return file.read()
+        return actual_file_name.read_text()
 
     def load_all_characters(self) -> str:
-        characters = ""
-
-        directory = os.path.join(os.path.dirname(__file__), "data")
-        for filename in os.listdir(directory):
-            with open(os.path.join(directory, filename), "r") as file:
-                characters = characters + file.read()
-        return characters
+        return "".join(file.read_text() for file in (Path(__file__).parent / "data").iterdir())
 
     def load_recent_characters(self, max: int) -> List[str]:
         try:
-            with open(os.path.join(BaseDirectory.xdg_data_home, 'rofimoji', 'recent'), 'r') as file:
-                return file.read().strip().split('\n')[:max]
+            return (BaseDirectory.xdg_data_home / 'rofimoji' / 'recent').read_text().strip().split('\n')[:max]
```

Comment:
Here, too, `BaseDirectory.xdg_data_home` isn't a `Path` for me

Reply:
The same as above. :-(